iceberg/src/test/scala/org/apache/spark/sql/delta/ConvertToIcebergSuite.scala

```diff
@@ -123,13 +122,15 @@ class ConvertToIcebergSuite extends QueryTest with Eventually {
         withDefaultTablePropsInSQLConf {
           deltaSpark.range(10).write.format("delta")
             .option("path", testTablePath)
+            .option("delta.enableIcebergCompatV2", "true")
```

Comment:
I was unable to let the tests pass without this property.

Reply:
Same here! cc @lzlfred @scottsand-db @harperjiang